aspnetcore/security/authentication/customize-identity-model.md

```diff
@@ -342,15 +342,37 @@ Update *Pages/Shared/_LoginPartial.cshtml* and replace `IdentityUser` with `Appl
 @inject UserManager<ApplicationUser> UserManager
 ```
 
-Update *Areas/Identity/IdentityHostingStartup.cs*  or `Startup.ConfigureServices` and replace `IdentityUser` with `ApplicationUser`.
+Update *Areas/Identity/IdentityHostingStartup.cs* or `Startup.ConfigureServices` and replace `IdentityUser` with `ApplicationUser`.
+
+::: moniker range=">= aspnetcore-2.1 <= aspnetcore-5.0"
 
 ```csharp
-services.AddIdentity<ApplicationUser>()
-        .AddEntityFrameworkStores<ApplicationDbContext>()
-        .AddDefaultUI();
+services.AddDefaultIdentity<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = true)
```

Comment:
Why isn't this

```suggestion
services.AddDefaultIdentity<IdentityUser>(options => options.SignIn.RequireConfirmedAccount = true)
```

Reply:
In this step, the doc suggests that we replace IdentityUser with the user-defined ApplicationUser type